Lib/Inc/IPwmOutput.h

```diff
@@ -0,0 +1,25 @@
+/**
+ * @file IPwmOutput.h
+ *
+ */
+
+/*
+ * IPwmOutput.h
+ *
+ *  Created on: 20 Feb 2022
+ *      Author: salavat.magazov
+ */
+
+#ifndef INC_IPWMOUTPUT_H_
```

Comment:
I think it is because it is in an INC_ is included because it is in an Inc folder. If you create a header file in the Src foulder it has SRC_ before the name.

Reply:
fixed.